src/high-level/constructors.lisp

```diff
@@ -75,8 +75,18 @@ The tensor is specialized on SHAPE and TYPE."
            (tensor (make-tensor tensor-class shape :layout layout))
            (shape-length (length shape))
            (fill-value (coerce (or value 1) (element-type tensor))))
-      (loop :for i :below (reduce #'min shape)
-            :do (setf (apply #'tref tensor (make-list shape-length :initial-element i)) fill-value))
+      (cond
+        ((/= offset 0)
+         (assert (cl:= shape-length 2)
+                 ()
```

Comment:
Include `shape` here so that it can be inspected / modified in a restart.

Reply:
Tho I s'pose it's not *that* helpful in an assert. 🤷‍♂

Reply:
Or maybe it is. 🤷‍♂

Reply:
I can't decide whether it is or not 🤷‍♂

Reply:
Doesn't hurt to include. Plus we're making fuller use of lisp's neat-o condition system.

Reply:
I agree, in this case, it doesn't hurt